### PR TITLE
.

### PR DESCRIPTION
**Pull request by ghost**
.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,126 +1,138 @@
...
-	text-align: center;
+#header-1 {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+#header-2 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
Comment (ziadomalik) [inline style.css:63]: thats like the two different header layouts ek showed me

**Comment by eepykate** [comment]
ill check this out later, but again please use tabs for indentation, rather than spaces. :(

**Comment by ziadomalik** [comment]
vs code reset the tabs to 4 spaces again after painstakingly trying to fix the tabs issue. forget it

**Comment by eepykate** [comment]
~~i'll fix it when i check it out later i guess.~~ nvm repo was yote

**Pull request closed without merging**